guide/interactions/slash-commands.md

```diff
@@ -196,27 +196,27 @@ client.on('interactionCreate', interaction => {
 });
 ```
 
-However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `CommandInteraction#isCommand()` method:
+However, not every interaction is a slash command (e.g. `MessageComponent`s). Make sure to only receive slash commands by making use of the `Interaction#isChatInputCommand()` method:
 
 ```js {2}
 client.on('interactionCreate', interaction => {
-	if (!interaction.isCommand()) return;
+	if (!interaction.isChatInputCommand()) return;
 	console.log(interaction);
 });
 ```
 
 ### Responding to a command
 
 There are multiple ways of responding to a slash command, each of these are covered in the following segments.
-The most common way of sending a response is by using the `CommandInteraction#reply()` method:
+The most common way of sending a response is by using the `Interaction#reply()` method:
 
 ::: warning
-Initially an interaction token is only valid for three seconds, so that's the timeframe in which you are able to use the `CommandInteraction#reply()` method. Responses that require more time ("Deferred Responses") are explained later in this page.
+Initially an interaction token is only valid for three seconds, so that's the timeframe in which you are able to use the `Interaction#reply()` method. Responses that require more time ("Deferred Responses") are explained later in this page.
```

Comment:
```suggestion
Initially an interaction token is only valid for three seconds, so that's the timeframe in which you are able to use the `BaseInteraction#reply()` method. Responses that require more time ("Deferred Responses") are explained later in this page.
```